jerry-core/parser/js/js-lexer.c

```diff
@@ -1544,7 +1544,7 @@ void
 lexer_next_token (parser_context_t *context_p) /**< context */
 {
   size_t length;
-
+  lexer_token_type_t type = context_p->token.type;
```

Comment:
`lexer_next_token` should not rdepend on this value

Reply:
So any suggestion?

Reply:
I think this is correctly parsed as number literal, and the issue should be handled where the number literal is accepted.